CONTRIBUTING.md

```diff
@@ -49,3 +49,15 @@ Now several cases are possible:
    Choose the start branch (the one you want to send your changes to) and the end branch (the one you want to send your changes to).
    Put an understandable name, click on `Assign Yourself` on the right,
    At the bottom of the description, note the id of your github issue like this: `Related issues: #1` or `Close #1`.
+
+## Create new Release
+
+To publish a new release you need to the repository, you need to be sure to be on the latest version. so you can use this commad : 
```

Comment:
> so you can use this commad :

A typo in "commad". Should be "command".  A redundant space before a colon symbol.

Also, "so" should be capitalized.

Reply:
Also the first sentence could be rewritten like:

> Before releasing a new version make sure you have the local repository up-to-date.

This explains the command below more clear, as the command actually makes your repository up-to-date.